docs/en/latest/plugins/cors.md

```diff
@@ -23,35 +28,36 @@ title: cors
+| allow_origins_by_regex    | array   | False    | nil     | Regex to match with origin for enabling CORS. For example, `[".*\.test.com"]` can match all subdomain of `test.com`.                                                                                                                                                                                                                                                                               |
+| allow_origins_by_metadata | array   | False    | nil     | Origins to enable CORS referenced from `allow_origins` set in the Plugin metadata. For example, if `"allow_origins": {"EXAMPLE": "https://example.com"}` is set in the Plugin metadata, then `["EXAMPLE"]` can be used to allow CORS on the origin `https://example.com`.                                                                                                                          |
+
+:::info IMPORTANT
+
+The `allow_credential` attribute is sensitive and must be used carefully. If set to `true` the default value `*` of the other attributes will be invalid and they should be specified explicitly. When using `**` you are vulnerable to security risks like CSRF. Make sure that this meets your security levels before using it.
+
+:::
 
 ## Metadata
 
-| Name          | Type    | Requirement | Default | Valid | Description                                                            |
-| -----------   | ------  | ----------- | ------- | ----- | ---------------------------------------------------------------------- |
-| allow_origins | object  | optional    |         |       | A map from origin reference to allowed origins; its key is the reference used by `allow_origins_by_metadata` and its value is a string equivalent to `allow_origins` in plugin attributes  |
+| Name          | Type   | Required | Description                                                                                                                                                                                             |
+|---------------|--------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| allow_origins | object | False    | A map with origin reference and allowed origins. The keys in the map are used in the attribute `allow_origins_by_metadata` and the value are equivalent to the `allow_origins` attribute of the Plugin. |
 
-## How To Enable
+## Enabling the Plugin
 
-Create a `Route` or `Service` object and configure `cors` plugin.
+You can enable the Plugin on a specific Route or Service:
 
 ```shell
 curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```

Comment:
For commands that are too long, should we format them? or keep it?
I think formatting long commands helps the user to see all the commands, otherwise he also needs to slide the scroll bar. Maybe we need to do this in all documents.

Reply:
Yes, I have kept it as it is on all docs. Maybe we can create a separate issue to ensure better formatting.